packages/example/vite.config.js

```diff
@@ -0,0 +1,6 @@
+export default {
```

Comment:
Not sure we really want this file - might cause more confusion than anything long term. Probably makes more sense to add it to `e2e/config`

Reply:
Yea, it would be nice to add some e2e tests for this (even as a part of some other existing tests).

Reply:
Moved it to `e2e/config`!